Cargo.toml

```
@@ -14,6 +14,7 @@ pedantic = { level = "warn", priority = -1 }
 unwrap_used = "warn"
 
 [dependencies]
+bullet_stream = "0.3.0"
```

(To match the existing style; cargo already uses the versions as a lower bound etc)

```suggestion
bullet_stream = "0.3"
```